src/main/java/org/folio/rest/support/EffectiveCallNumberComponentsUtil.java

```diff
@@ -35,12 +40,11 @@ public static void calculateAndSetEffectiveShelvingOrder(Item item) {
           .map(StringUtils::trim)
           .collect(Collectors.joining(" "))
       );
-//      String suffixValue = Objects.toString(item.getEffectiveCallNumberComponents().getSuffix(), "").trim();
-      String suffixValue =
-        Objects.toString(Optional.ofNullable(item.getEffectiveCallNumberComponents())
-          .orElse(new EffectiveCallNumberComponents()).getSuffix(), "")
-          .trim();
 
+      String suffixValue =
+          Objects.toString(Optional.ofNullable(item.getEffectiveCallNumberComponents())
+              .orElse(new EffectiveCallNumberComponents()).getSuffix(), "")
+              .trim();
```

Comment:
Why has the indentation of these lines changed?

Reply:
Seems it related disabled eclipse formatting plugin. I'll compare it with already merged code to make correction until formatting be fixed.